README.Rmd

```diff
@@ -13,13 +13,18 @@ knitr::opts_chunk$set(
 )
 ```
 
+```{r srr-tags, eval=FALSE, echo=FALSE}
+#' @srrstats {G1.2} Lifecycle badge added in the README
```

Comment:
Should we also explicitly add asentence on the current level of development (from the [docs](https://stats-devguide.ropensci.org/standards.html#documentation) the follwing might fit: `In a stable state of development, with some degree of active subsequent
      development as envisioned by the primary authors`).

Reply:
Added in the Contributing section